bot.js

```diff
@@ -5,31 +5,36 @@ var auth = require('./auth.json');
 // Configure logger settings
 logger.remove(logger.transports.Console);
 logger.add(logger.transports.Console, {
-    colorize: true
+  colorize: true
 });
 
 logger.level = 'debug';
 
 // Initialize Discord Bot
 var bot = new Discord.Client({
-   token: auth.token,
-   autorun: true,
+  token: auth.token,
+  autorun: true,
 });
 
 bot.on('ready', function (evt) {
-    logger.info('Connected');
-    logger.info('Logged in as: ');
-    logger.info(bot.username + ' - (' + bot.id + ')');
+  logger.info('Connected');
+  logger.info('Logged in as: ');
+  logger.info(bot.username + ' - (' + bot.id + ')');
 });
 
 bot.on('message', function (user, userID, channelID, message, evt) {
-    switch(message) {
-        case 'Ассаляму аляйкум':
-            bot.sendMessage({
-                to: channelID,
-                message: 'Уа аляйкум салям',
-            });
+  let regexp = /\!thanks/gmi;
```

Comment:
This regexp is good, but still you have a number of corner cases:
1. What if someone wrote `!thankswhatever`, your bot will catch it, but it shouldn't.
2. What if someone wrote `... end of the sentence!thanks Mike for ...`. Your bot will also catch it, but again, I don't think we want it.